### ISSUE TITLE
Some questions about your experiment

### ISSUE THREAD

**Issue by lixiangMindSpore** [jizhishutong/YOLOU#2]
![image](https://user-images.githubusercontent.com/78945582/180917869-ca6668f5-a99b-4419-b511-d305469e9631.png)
Your experiment is based on COCO dataset? If it is true, some data is not consistent the official data.

**Comment by ChaucerG** [comment]
Yes, but some results are not up to the official accuracy, we are still working on it.

**Comment by Mobu59** [comment]
Some data seems to be different from the official one? For example, the parameters and flops of yolox-s, the official ones are 9.0 and 26.8, while your experiment is 8.1 and 10.8, what is the reason for the difference?

**Comment by jizhishutong** [comment]
You can pay attention to YOLOXS.yaml, where Backbone and Neck use the same as YOLOv5s, so compared to YOLOv5s, FLOPS and  params are only slightly increased.


---- Replied Message ----
| From | ***@***.***> |
| Date | 7/27/2022 10:21 |
| To | ***@***.***> |
| Cc | ***@***.***> |
| Subject | Re: [jizhishutong/YOLOU] Some questions about your experiment (Issue #2) |

Some data seems to be different from the official one? For example, the parameters and flops of yolox-s, the official ones are 9.0 and 26.8, while your experiment is 8.1 and 10.8, what is the reason for the difference?
